specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2019-10-01-preview/WebTestResults_API.json

```diff
@@ -0,0 +1,976 @@
+					{
+						"$ref": "#/parameters/WebtestNameParameter"
+					},
+					{
+						"$ref": "#/parameters/PosixTimeStampParameter"
+					},
+					{
+						"$ref": "#/parameters/TestSuccessfulParameter"
+					},
+					{
+						"$ref": "#/parameters/DownloadAsParameter"
+					},
+					{
+						"$ref": "#/parameters/OffsetParameter"
+					}
+				],
+				"responses": {
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					},
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/TestResultFileResponse"
+						}
+					}
+				},
+				"x-ms-examples": {
+					"Get Metadata for Operation": {
+						"$ref": "./examples/WebTestResultsTestResultFile.json"
+					}
+				}
+			}
+		},
+		"/subscriptions/{subscriptionId}/resoucegroups/{resourceGroupName}/providers/microsoft.insights/webtests/{webtestName}/popncus/testresultfile": {
+			"get": {
+				"tags": [
+					"FindWebTestResult"
+				],
+				"operationId": "testresultfile_ncus",
+				"description": "Returns a file test result for the matching test",
+				"parameters": [
+					{
+						"$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+					},
+					{
+						"$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+					},
+					{
+						"$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+					},
+					{
+						"$ref": "#/parameters/WebtestNameParameter"
+					},
+					{
+						"$ref": "#/parameters/PosixTimeStampParameter"
+					},
+					{
+						"$ref": "#/parameters/TestSuccessfulParameter"
+					},
+					{
+						"$ref": "#/parameters/DownloadAsParameter"
+					},
+					{
+						"$ref": "#/parameters/OffsetParameter"
+					}
+				],
+				"responses": {
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "#/definitions/ErrorResponse"
+						}
+					},
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/TestResultFileResponse"
+						}
+					}
+				},
+				"x-ms-examples": {
+					"Get Metadata for Operation": {
+						"$ref": "./examples/WebTestResultsTestResultFile.json"
+					}
+				}
+			}
+		}
+	},
+	"definitions": {
+		"ErrorResponse": {
+			"description": "Describes the format of Error response.",
+			"type": "object",
+			"properties": {
+				"code": {
+					"description": "Error code",
+					"type": "string"
+				},
+				"message": {
+					"description": "Error message indicating why the operation failed.",
+					"type": "string"
+				}
+			}
+		},
+		"TestResultFileResponse": {
+			"description": "Test result",
+			"type": "object",
+			"properties": {
+				"data": {
+					"description": "File contents",
+					"type": "string"
+				},
+				"nextLink": {
```

Comment:
What would be in this nextLink?  This isn't a collection response so the SDKs aren't going to follow this nextLink as-is.  When would this appear?